### PR TITLE
Add more detailed notice for environment variables

### PR DESCRIPTION
**Pull request by JanMalte**

### CODE REVIEW

**Review by fritzmg — COMMENTED** [review]
Not really sure this is necessary as nothing bad will happen if you use both.

**Comment by JanMalte** [comment]
I just translated and added the text from the english version:
https://docs.contao.org/dev/reference/config/#environment-variables-for-the-contao-managed-edition

File: docs/manual/system/settings.de.md [hunk 1]
```diff
@@ -548,7 +548,9 @@ Umgebungsvariablen sind Variablen, die auf Betriebssystem-Ebene, pro Benutzer od
 Die Variablen werden in der Datei `.env` definiert und diese Datei muß sich im Hauptverzeichnis der Contao-Installation befinden. Die Nutzung und Bezeichnung der folgenden Variablen sind vorgegeben. Du kannst aber auch beliebige Variablen definieren und dann z. B. in der `config.yml` referenzieren. Falls eine zusätzliche `.env.local` Datei im gleichen Verzeichnis existiert, wird diese automatisch verwendet.
 
 {{% notice info %}}
-Einige der Umgebungsvariablen, wie `APP_SECRET`, `DATABASE_URL` und `MAILER_DSN` ersetzen ihre jeweiligen Gegenstücke in der `config/parameters.yaml`.
+Einige der Umgebungsvariablen, wie `APP_SECRET`, `DATABASE_URL` und `MAILER_DSN` ersetzen ihre jeweiligen Gegenstücke 
+in der `config/parameters.yaml` und daher solltest du diese Parameter nicht verwendet werden, wenn du stattdessen die Umgebungsvariable
```
Comment (netzarbeiter) [inline docs/manual/system/settings.de.md]:
```suggestion
in der `config/parameters.yaml` und daher sollten diese Parameter nicht verwendet werden, wenn du stattdessen die Umgebungsvariable 
```

**Review by netzarbeiter — CHANGES_REQUESTED** [review]
Thank you, Jan

**Comment by netzarbeiter** [comment]
I am closing this ticket because of the following [PR](https://github.com/contao/docs/pull/1133).

**Pull request closed without merging**